tests/pages/firefox/android.py

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class AndroidPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/android'
+
+    _customize_locator = (By.CSS_SELECTOR, '#customize-accordion > .customize-section')
+    _next_button_locator = (By.ID, 'customize-next')
+    _previous_button_locator = (By.ID, 'customize-prev')
+
+    @property
+    def customize_sections(self):
+        return [CustomizeSection(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._customize_locator)]
+
+    def show_next_customize_section(self):
+        section = next(s for s in self.customize_sections if s.is_displayed)
+        self.scroll_element_into_view(self._next_button_locator).click()
+        self.wait.until(lambda s:
+            self.is_next_enabled and self.is_previous_enabled and not section.is_displayed)
+
+    def show_previous_customize_section(self):
+        section = next(s for s in self.customize_sections if s.is_displayed)
```

Comment:
You could abstract this into a `current_customize_section` property for less duplication.